src/tests/JIT/Regression/clr-x64-JIT/v2.1/b173569/b173569.ilproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk.IL">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <CLRTestPriority>1</CLRTestPriority>
     <!-- Test unsupported outside of windows -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
It might be better to insert the RequireProcessIsolation line below the CLRTestTargetUnsupported line as otherwise it can be separated from an explanatory comment as happened here.

Reply:
Having said that, in this particular case the explanatory comment has arguably little value beyond what the script code actually says.